### DISCUSSION TITLE
importing resolveConfig causes an error after upgrading to TW 2.2.2

### DISCUSSION THREAD

**Discussion by rhysemmerson** [tailwindlabs/tailwindcss#4729] [Help]
After upgrading to tailwindcss 2.2.2 I'm getting a build error, it happens when importing `tailwindcss/resolveConfig` 

```
ERROR in ./node_modules/tailwindcss/lib/plugins/preflight.js 8:33-46
Module not found: Error: Can't resolve 'fs' in '/Users/rhys/Code/account/node_modules/tailwindcss/lib/plugins'
```

Downgrading to version 2.1.4 fixes it.

Is there a better way to inject the config into the client side js?

**Comment by simonswiss** [comment]
Hey!

Just tried this with a NextJS 11 project, and getting another error with Tailwind CSS v2.2:

```sh
error - ./node_modules/modern-normalize/modern-normalize.css
Global CSS cannot be imported from within node_modules.
Read more: https://nextjs.org/docs/messages/css-npm
Location: node_modules/tailwindcss/lib/plugins/preflight.js
```

Downgrading to `v2.1.x` fixes the issue as well.

~I'll open an issue and we'll take a look at this 👍~

**EDIT**: An issue is [already opened](https://github.com/tailwindlabs/tailwindcss/issues/4681) for this.
